content/news/041/index.md

```diff
@@ -468,6 +468,46 @@ it got a lot of new functionality and improved existing:
 
 ## Tooling Updates
 
+### [Graphite][graphite-website]
+
+[
+![Graphite logo](graphite-vector-demo.png)
+_Video: 3-minute timelapse of a vector art piece illustrated in Graphite_
+][graphite-vector-demo]
+
+Graphite ([website][graphite-website], [GitHub][graphite-repo],
+[Discord][graphite-discord], [Twitter][graphite-twitter]) is a free,
+in-development raster and vector 2D graphics editor based around a Rust-powered
+node graph compositing engine.
+
+December's [sprint 21][graphite-sprint-21] introduces:
+
+- Chain reaction: The Imaginate feature, an AI image generation workflow
+  powered by [Stable Diffusion][graphite-stable-diffusion], becomes a node.
+  Chain together a sequence of fine-tuned generation steps. And explore ideas
+  by branching the graph into new creative directions.
+- Node nurturing: New features provide polish to the node graph. Nodes can be
+  copy/pasted, hidden, previewed, and linked more easily.
+- Bugs, begone!: A major effort to improve editor usability fixes dozens of
+  bugs and paper cuts. Boolean shape operations now crash less frequently, the
+  UI no longer slows down badly over time, and undo history is finally fixed.
+
+Stay tuned for the imminent Alpha Milestone 2 release and progress converting
+existing features into nodes.
+
+Open the [Graphite editor][graphite-editor] in your browser to give it a try
+and share your creations with #MadeWithGraphite on Twitter.
+
+[graphite-vector-demo]: https://www.youtube.com/watch?v=JgJvAHQLnXA
+  "Graphite Vector Editing: \"Commander Basstronaut\" Artwork (25x Timelapse)"
+[graphite-website]: https://graphite.rs
```

Comment:
I think this line should be removed?

```suggestion
[graphite-vector-demo]: https://www.youtube.com/watch?v=JgJvAHQLnXA
[graphite-website]: https://graphite.rs
```

Reply:
It's MD's way of giving a title to the link - IIRC, should show up on a cursor hover event, for example.

Reply:
As @ozkriff answered, this is the title text for the link. I've confirmed the syntax works.